distrho/src/DistrhoUILV2.cpp

```diff
@@ -289,7 +289,7 @@ class UiLv2
 
         // reserve atom space
         const size_t atomSize = sizeof(LV2_Atom) + msgSize;
-        char         atomBuf[atomSize];
+        char*        atomBuf = (char*)alloca(atomSize);
```

Comment:
why is this alloca and not malloc?
I understand it uses the stack, mostly wondering when one picks alloca vs malloc (previously in the string case you had malloc)